deps.edn

```diff
@@ -1,13 +1,23 @@
-{:paths ["src" "resources"]
+{:paths ["src" "resources" "notebooks"]
```

Comment:
we should add a `:dev` alias that has notebooks in it and the `user.clj` in a `dev` rather than `src` directory